deb/deb_test.go

```diff
@@ -1084,3 +1084,21 @@ func TestEmptyButRequiredDebFields(t *testing.T) {
 	err := Default.Package(item, &deb)
 	require.NoError(t, err)
 }
+
+func TestArches(t *testing.T) {
+	info := exampleInfo()
+
+	for k := range archToDebian {
+		t.Run(k, func(t *testing.T) {
+			info.Arch = k
+			info = ensureValidArch(info)
+			require.Equal(t, archToDebian[k], info.Arch)
+		})
+	}
+
+	t.Run("override", func(t *testing.T) {
+		info.Deb.Arch = "foo64"
```

Comment:
I'd add `info.Arch = "amd64"` or something like this before this line. Otherwise this test depends on whatever value `info.Arch` was set to in the previous subtests. This also makes it more clear in what way the override behavior is tested.

Reply:
fixed it to use a new exampleInfo on every subtest :)